Prop type errors on RTCView when using in TypeScript

<!--
If you have a question then please ask it on our community Discourse: https://react-native-webrtc.discourse.group/  
otherwise if you wanted to report a bug then you are in the right place!  

Please include as much information as possible and make the issue title as descriptive as you can.
-->

#### Expected Behavior
Using the various props on RTCView (style, streamURL, objectFit, etc.) should work without type errors in TypeScript.

#### Observed Behavior
An error is raised when running `tsc`:

```
App.tsx:50:11 - error TS2322: Type '{ style: { flex: number; }; streamURL: string; objectFit: any; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<unknown, {}, any> & Readonly<NativeMethods>> & Readonly<...>'.
  Property 'style' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<unknown, {}, any> & Readonly<NativeMethods>> & Readonly<...>'.

50           style={{ flex: 1 }}
             ~~~~~
```

#### Steps to reproduce the issue
Simply use RTCView in a typescript file and pass any props to it like so:

```
      <RTCView
          style={styles.videoView}
          streamURL={''}
          objectFit={'cover'}
        />
```


#### Platform Information

```
    "react": "18.1.0",
    "react-native": "0.70.5",
    "react-native-webrtc": "^106.0.0-beta.6"

    "typescript": "^4.8.3"
```

--- a/src/RTCView.ts
+++ b/src/RTCView.ts
@@ -1,5 +1,4 @@
-
-import { requireNativeComponent } from 'react-native';
+import { requireNativeComponent, ViewProps } from 'react-native';
 
 /**
  * Native prop validation was removed from RN in:
@@ -8,53 +7,58 @@ import { requireNativeComponent } from 'react-native';
  * So we list them here for documentation purposes.
  */
 
-/**
- * Indicates whether the video specified by {@link #streamURL} should be
- * mirrored during rendering. Commonly, applications choose to mirror the
- * user-facing camera.
- *
- * mirror: boolean
- */
+interface RTCVideoViewProps extends ViewProps {
+  /**
+   * Indicates whether the video specified by {@link #streamURL} should be
+   * mirrored during rendering. Commonly, applications choose to mirror the
+   * user-facing camera.
+   *
+   * mirror: boolean
+   */
+  mirror?: boolean;
 
-/**
- * In the fashion of
- * https://www.w3.org/TR/html5/embedded-content-0.html#dom-video-videowidth
- * and https://www.w3.org/TR/html5/rendering.html#video-object-fit,
- * resembles the CSS style object-fit.
- *
- * objectFit: 'contain' | 'cover'
- */
+  /**
+   * In the fashion of
+   * https://www.w3.org/TR/html5/embedded-content-0.html#dom-video-videowidth
+   * and https://www.w3.org/TR/html5/rendering.html#video-object-fit,
+   * resembles the CSS style object-fit.
+   *
+   * objectFit: 'contain' | 'cover'
+   */
+  objectFit?: 'contain' | 'cover';
 
-/**
- * URL / id of the stream that should be rendered.
- *
- * streamURL: string
- */
+  /**
+   * URL / id of the stream that should be rendered.
+   *
+   * streamURL: string
+   */
+  streamURL: string;
+  /**
+   * Similarly to the CSS property z-index, specifies the z-order of this
+   * RTCView in the stacking space of all RTCViews. When RTCViews overlap,
+   * zOrder determines which one covers the other. An RTCView with a larger
+   * zOrder generally covers an RTCView with a lower one.
+   *
+   * Non-overlapping RTCViews may safely share a z-order (because one does not
+   * have to cover the other).
+   *
+   * The support for zOrder is platform-dependent and/or
+   * implementation-specific. Thus, specifying a value for zOrder is to be
+   * thought of as giving a hint rather than as imposing a requirement. For
+   * example, video renderers such as RTCView are commonly implemented using
+   * OpenGL and OpenGL views may have different numbers of layers in their
+   * stacking space. Android has three: a layer bellow the window (aka
+   * default), a layer bellow the window again but above the previous layer
+   * (aka media overlay), and above the window. Consequently, it is advisable
+   * to limit the number of utilized layers in the stacking space to the
+   * minimum sufficient for the desired display. For example, a video call
+   * application usually needs a maximum of two zOrder values: 0 for the
+   * remote video(s) which appear in the background, and 1 for the local
+   * video(s) which appear above the remote video(s).
+   *
+   * zOrder: number
+   */
+  zOrder?: number;
+}
 
-/**
- * Similarly to the CSS property z-index, specifies the z-order of this
- * RTCView in the stacking space of all RTCViews. When RTCViews overlap,
- * zOrder determines which one covers the other. An RTCView with a larger
- * zOrder generally covers an RTCView with a lower one.
- *
- * Non-overlapping RTCViews may safely share a z-order (because one does not
- * have to cover the other).
- *
- * The support for zOrder is platform-dependent and/or
- * implementation-specific. Thus, specifying a value for zOrder is to be
- * thought of as giving a hint rather than as imposing a requirement. For
- * example, video renderers such as RTCView are commonly implemented using
- * OpenGL and OpenGL views may have different numbers of layers in their
- * stacking space. Android has three: a layer bellow the window (aka
- * default), a layer bellow the window again but above the previous layer
- * (aka media overlay), and above the window. Consequently, it is advisable
- * to limit the number of utilized layers in the stacking space to the
- * minimum sufficient for the desired display. For example, a video call
- * application usually needs a maximum of two zOrder values: 0 for the
- * remote video(s) which appear in the background, and 1 for the local
- * video(s) which appear above the remote video(s).
- *
- * zOrder: number
- */
-
-export default requireNativeComponent('RTCVideoView');
+export default requireNativeComponent<RTCVideoViewProps>('RTCVideoView');

